### PR TITLE
Add js css file

### PR DESCRIPTION
**Pull request by nguyenkhacduyngoc**
Thêm js, css cho frontend

### CODE REVIEW

File: resources/assets/js/frontend/custom.js [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+    	smartSpeed:500,
+		loop: true,
+		items : 1,
+		nav : false,
+		margin: 0,
+		responsiveClass: true,
+		dots: true,
+		responsive: {
+			0: {
+				items: 1
```
Comment (trinhductoan) [inline resources/assets/js/frontend/custom.js]: indent, 4 space thay cho 1 tab

File: resources/assets/js/frontend/form.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	// Get the form.
+	var form = $('#ajax-contact');
+
+	// Get the messages div.
+	var formMessages = $('#form-messages');
+
+	// Set up an event listener for the contact form.
+	$(form).submit(function(e) {
+		// Stop the browser from submitting the form.
+		e.preventDefault();
```
Comment (trinhductoan) [inline resources/assets/js/frontend/form.js]: indent, 4 space thay cho 1 tab

File: resources/assets/js/frontend/jquery.sticky.js [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+            s.stickyElement.parent().removeClass(s.className);
+            s.stickyElement.trigger('sticky-end', [s]);
+            s.currentTop = null;
+          }
+        }
+        else {
+          var newTop = documentHeight - s.stickyElement.outerHeight()
+            - s.topSpacing - s.bottomSpacing - scrollTop - extra;
+          if (newTop < 0) {
+            newTop = newTop + s.topSpacing;
```
Comment (trinhductoan) [inline resources/assets/js/frontend/jquery.sticky.js]: indent, 4 space thay cho 1 tab

File: resources/assets/js/frontend/map.js [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+            maxWidth: 400
+        });
+
+        marker.addListener('click', function () {
+            infowindow.open(map, marker);
+        });
+
+        var styles = 
+        [
+    {
```
Comment (trinhductoan) [inline resources/assets/js/frontend/map.js]: indent, 4 space thay cho 1 tab

File: resources/assets/sass/frontend/magnific-popup.css [hunk 5]
```diff
@@ -0,0 +1,351 @@
...
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  left: 0;
+  top: 0;
+  padding: 0 8px;
+  box-sizing: border-box; }
+
+.mfp-container:before {
+  content: '';
```
Comment (trinhductoan) [inline resources/assets/sass/frontend/magnific-popup.css]: conventions

File: resources/assets/sass/frontend/responsive.css [hunk 6]
```diff
@@ -0,0 +1,143 @@
+/* Responsive Styles 
+–––––––––––––––––––––––––––––––––––––––––––––––––– */
+
+@media only screen and (max-width: 1199px) {
+	.allnews .follow-widget ul li,
+	.news-details .follow-widget ul li,
+	.catagory .follow-widget ul li {
```
Comment (trinhductoan) [inline resources/assets/sass/frontend/responsive.css]: indent, 4 space thay cho 1 tab

**Comment by trinhductoan** [comment]
commented

File: resources/assets/js/frontend/custom.js [hunk 7]
```diff
@@ -0,0 +1,274 @@
...
+		year: 2018,
+		month: 2,
+		day: 1,
+		hour: 7,
+		minute: 30
+	});
+
+	/***** Back To Top *****/
+	$(window).scroll(function(){
+	if($(this).scrollTop()>500){
```
Comment (BunnyPi04) [inline resources/assets/js/frontend/custom.js]: convention

File: resources/assets/js/frontend/custom.js [hunk 8]
```diff
@@ -0,0 +1,274 @@
...
+	    $(".back-to-top i").fadeIn();
+	}else{
```
Comment (BunnyPi04) [inline resources/assets/js/frontend/custom.js]: convention

File: resources/assets/js/frontend/custom.js [hunk 9]
```diff
@@ -0,0 +1,274 @@
...
+	    $("html, body").animate({scrollTop:0}, 900);
+	});
+
+	/***** Preloader *****/
+	var $preloader = $('#page-preloader'),
+    $spinner   = $preloader.find('.spinner-loader');
+    $spinner.fadeOut();
+    $preloader.delay(50).fadeOut('slow');
+
+});
```
Comment (BunnyPi04) [inline resources/assets/js/frontend/custom.js]: dòng trống

File: resources/assets/js/frontend/custom.js [hunk 10]
```diff
@@ -0,0 +1,274 @@
...
+	    $(".back-to-top i").hide();
+	}
+	});
+	$(".back-to-top i").on('click',function(){
+	    $("html, body").animate({scrollTop:0}, 900);
+	});
+
+	/***** Preloader *****/
+	var $preloader = $('#page-preloader'),
+    $spinner   = $preloader.find('.spinner-loader');
```
Comment (BunnyPi04) [inline resources/assets/js/frontend/custom.js]: convention

File: resources/assets/js/frontend/jquery.sticky.js [hunk 11]
```diff
@@ -0,0 +1,287 @@
...
+        // AMD. Register as an anonymous module.
+        define(['jquery'], factory);
+    } else if (typeof module === 'object' && module.exports) {
+        // Node/CommonJS
+        module.exports = factory(require('jquery'));
+    } else {
+        // Browser globals
+        factory(jQuery);
+    }
+}(function ($) {
```
Comment (BunnyPi04) [inline resources/assets/js/frontend/jquery.sticky.js]: convention

File: resources/assets/js/frontend/jquery.sticky.js [hunk 12]
```diff
@@ -0,0 +1,287 @@
...
+    var slice = Array.prototype.slice; // save ref to original slice()
+    var splice = Array.prototype.splice; // save ref to original slice()
+
+  var defaults = {
```
Comment (BunnyPi04) [inline resources/assets/js/frontend/jquery.sticky.js]: convention

File: resources/assets/js/frontend/jquery.sticky.js [hunk 13]
```diff
@@ -0,0 +1,287 @@
...
+            if (s.getWidthFrom) {
+                newWidth = $(s.getWidthFrom).width() || null;
+            } else if (s.widthFromWrapper) {
+                newWidth = s.stickyWrapper.width();
+            }
+            if (newWidth == null) {
+                newWidth = s.stickyElement.width();
+            }
+            s.stickyElement
+              .css('width', newWidth)
```
Comment (BunnyPi04) [inline resources/assets/js/frontend/jquery.sticky.js]: indent

**Pull request merged**